ios/Classes/InterstitialAd.swift

```diff
@@ -81,7 +81,7 @@ extension InterstitialAd: GADInterstitialDelegate {
 
     /// Tells the delegate the interstitial is to be animated off the screen.
     func interstitialWillDismissScreen(_ ad: DFPInterstitial) {
-        interstitialAd = DFPInterstitial(adUnitID: ad.adUnitID)
+        interstitialAd = DFPInterstitial(adUnitID: ad.adUnitID!)
```

Comment:
A compilation error has occurred due to this change.   

What version of Google-Mobile-Ads-SDK do you use?

Reply:
I'm using the one that got installed by default by adding the Flutter library. Here's my Podfile.lock

`  - Google-Mobile-Ads-SDK (7.42.2):
    - GoogleAppMeasurement (~> 5.7)
  - GoogleAppMeasurement (5.8.0):
    - GoogleUtilities/AppDelegateSwizzler (~> 5.2)
    - GoogleUtilities/MethodSwizzler (~> 5.2)
    - GoogleUtilities/Network (~> 5.2)
    - "GoogleUtilities/NSData+zlib (~> 5.2)"
    - nanopb (~> 0.3)
  - GoogleUtilities/AppDelegateSwizzler (5.5.0):
    - GoogleUtilities/Environment
    - GoogleUtilities/Logger
    - GoogleUtilities/Network
  - GoogleUtilities/Environment (5.5.0)
  - GoogleUtilities/Logger (5.5.0):
    - GoogleUtilities/Environment
  - GoogleUtilities/MethodSwizzler (5.5.0):
    - GoogleUtilities/Logger
  - GoogleUtilities/Network (5.5.0):
    - GoogleUtilities/Logger
    - "GoogleUtilities/NSData+zlib"
    - GoogleUtilities/Reachability
  - "GoogleUtilities/NSData+zlib (5.5.0)"
  - GoogleUtilities/Reachability (5.5.0):
    - GoogleUtilities/Logger`

Reply:
Oh, one other note, the project is using Swift 4, again the default that Flutter set my project up with.

Reply:
You're right. It seems that it has changed from Google-Mobile-Ads-SDK v7.41.

Thank you for your contribution.